parts/linux/cloud-init/artifacts/cse_cmd.sh

```diff
@@ -12,7 +12,7 @@ done;
 {{GetInitAKSCustomCloudFilepath}} >> /var/log/azure/cluster-provision.log 2>&1;
 {{end}}
 ADMINUSER={{GetParameter "linuxAdminUsername"}}
-CONTAINERD_VERSION={{GetParameter "containerdVersion"}}
+CONTAINERD_VERSION="1.3.4"
```

Comment:
1.3.7

Reply:
do we need the extra "" ? We can actually remove this line

Reply:
removing